src/azure-cli/azure/cli/command_modules/databoxedge/aaz/latest/databoxedge/device/user/_list.py

```diff
@@ -55,7 +55,7 @@ def _build_arguments_schema(cls, *args, **kwargs):
         )
         _args_schema.filter = AAZStrArg(
             options=["--filter"],
-            help="Specify $filter='Type eq <type>' to filter on user type property",
+            help="Specify `$filter='Type eq <type>'` to filter on user type property",
```

Comment:
Is this change synced with aaz repo?

Reply:
Yes, aaz has updated in this pr: https://github.com/Azure/aaz/pull/655